configure.ac

```diff
@@ -88,6 +88,12 @@ AC_ARG_ENABLE([mining],
   [enable_mining=$enableval],
   [enable_mining=yes])
 
+AC_ARG_ENABLE([rust],
+  [AS_HELP_STRING([--enable-rust],
```

Comment:
(In an engineering meeting we decided not to make ``--disable-rust`` the default, but to add it to the User Guide instead. #2191)